### PR TITLE
Added Navbar to Routes

### PR DESCRIPTION
**Pull request by gunbux**
### Changes made
- Added Navbar to routes so that implementation of navbar isn't needed in individual pages

### CODE REVIEW

**Comment by gunbux** [comment]
[contacts page](https://trello.com/c/5L55A75o/20-contacts-page)

File: src/components/styles/NavBar.styled.ts [hunk 1]
```diff
@@ -4,5 +4,10 @@ export const MainContainer = styled.div`
   width: 65%;
   display: flex;
   margin-left: 10%;
-  justify-content: space-evenly;
+  justify-content: space-between;
+`
+
+export const NavContainer = styled.div`
+  width: 100%;
```
Comment (hui444) [inline src/components/styles/NavBar.styled.ts:11]: might want to use 100vw instead? not sure if this might cause issues

**Review by hui444 — APPROVED** [review]
lgtm!

File: src/components/styles/NavBar.styled.ts (hunk 1 above)
Comment (gunbux) [reply src/components/styles/NavBar.styled.ts]: i was under the impression they did the same things, but will change

**Pull request merged**